.vscode/launch.json

```diff
@@ -0,0 +1,65 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Probably doesn't matter to much but this won't be valid json. Might be worth putting this or something similar in the readme instead?

Reply:
This was autogenerated by VSCode. Happy to remove :)